CityPuzzle/GameEntryPage.xaml

```diff
@@ -84,7 +84,7 @@
                 <ColumnDefinition Width="50" />
 
             </Grid.ColumnDefinitions>
-            <Button Image="settings.png" AbsoluteLayout.LayoutFlags="PositionProportional" Grid.Column="3"/>
+            <Button Image="settings.png" AbsoluteLayout.LayoutFlags="PositionProportional" Grid.Column="3"  Clicked="Settings_Clicked" CornerRadius="20"/>
```

Comment:
If this does not open settings then names and image should be changed accordingly

Reply:
I agree with that. I will create the settings page in the future with the option to log out.